tests/test_screen.py

```diff
@@ -936,6 +936,45 @@ def signal_check(self, screen):
         """Dummy callback for screen wrapper."""
         self.assertEqual(signal.getsignal(signal.SIGWINCH), screen._resize_handler)
 
+    def test_overflow_decorator(self):
+        """
+        Check underflow and overflow work as expected
```

Comment:
Could you please update the comment to explain that it is testing that the ScreenSession class works as a function decorator?